values.schema.json

```diff
@@ -38,11 +42,12 @@
       },
       "sql": {
         "type": "object", "additionalProperties": false,
-        "required": ["url", "username", "password"],
+        "required": ["url", "secretName", "secretUserField","secretPasswordField"],
         "properties": {
           "url": {"type": "string"},
-          "username": {"type": "string"},
-          "password": {"type": "string"}
+          "secretName": {"type": "string"},
```

Comment:
1. Adjust new fields names
    * `existingSecret`
    * `existingSecretUsernameKey`
    * `existingSecretPasswordKey`
1. Declare default values, corresponding to `values.yaml`

Reply:
Done